### ISSUE TITLE
Request: Player whitelist or blacklist

### ISSUE THREAD

**Issue by AndydeCleyre** [InputUsername/rescrobbled#21]
I don't want to scrobble from every player that sends mpris signals, so it would be helpful to read from the config file either a player whitelist or blacklist. In a little applet I use for displaying the currently played track, I use the following:

```python
PLAYER_BLACKLIST_REGEXP = r'Gwenview|plasma-browser-integration|mpv|chromium\.instance'
```

This is then checked against the *beginning* of the player's broadcasted name (as returned by `playerctl`) so that my applet can ignore anything from those players.

I think for rescrobbled, a whitelist probably makes more sense and would be simpler to implement (no real need for regex matching, just literal full matches).

**Comment by InputUsername** [comment]
Thank you for the suggestion! An optional whitelist sounds like a good idea to add.

**Comment by InputUsername** [comment]
I've implemented this in #23! Thanks again for the suggestion.
